### PR TITLE
Update for DT-as-monorepo

### PR DESCRIPTION
**Pull request by jakebailey**
WIP, will un-draft once the final format is defined.

### CODE REVIEW

File: lib/definitely-typed.ts [hunk 1]
```diff
@@ -83,39 +104,21 @@ async function getIndex(content: string, packageName: string): Promise<string> {
...
-            noImplicitThis: true,
-            strictFunctionTypes: true,
-            strictNullChecks: true,
-            baseUrl: "../",
-            typeRoots: ["../"],
-            types: [],
-            noEmit: true,
-            forceConsistentCasingInFileNames: true,
+        private: true,
+        name: packageName,
```
Comment (sandersn) [inline lib/definitely-typed.ts:110]: packagename is something like `a__b` for `@a/b`, but should be `@types/a__b`

**Review by sandersn — CHANGES_REQUESTED** [review]
This looks good except I think the package name is incorrect in package.json.

File: lib/definitely-typed.ts (hunk 1 above)
Comment (jakebailey) [reply lib/definitely-typed.ts:110]: Hm, seems like packageName here is the name of the non-types package, not the mangled name either.

```ts
const reg: Registry = JSON.parse(await loadString(`https://registry.npmjs.org/${packageName}`));
```

So, I need to mangle it and that's it.

Comment (jakebailey) [reply lib/definitely-typed.ts]: Updated, PTAL

**Review by sandersn — APPROVED** [review]

**Pull request merged**